style.css

```diff
@@ -55,3 +55,33 @@ section p {
   line-height: 1.5;
   max-width: 55ch;
 }
+.cards {
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  gap: 2rem;
+}
+.cards .card {
+  position: relative;
+  display: flex;
+  transition: all 1s ease-in-out;
+}
+
+.cards .card:hover {
+  transform: rotate3d(0, 1, 0.5, 1turn);
```

Comment:
I'm really impressed with the complex CSS here, nothing sticks out as being off here. Good work!